dss-core/src/main/java/io/github/ztkmkoo/dss/core/exception/handler/DssRestExceptionHandlerResolver.java

```diff
@@ -0,0 +1,74 @@
+package io.github.ztkmkoo.dss.core.exception.handler;
+
+import io.github.ztkmkoo.dss.core.actor.rest.entity.DssRestServiceResponse;
+import io.github.ztkmkoo.dss.core.actor.rest.service.DssRestActorService;
+import io.github.ztkmkoo.dss.core.exception.annotation.ExceptionHandler;
+import io.github.ztkmkoo.dss.core.exception.annotation.ServiceExceptionHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+public class DssRestExceptionHandlerResolver {
+
+    private static DssRestExceptionHandlerResolver dssRestExceptionHandlerResolver = new DssRestExceptionHandlerResolver();
+
+    public static DssRestExceptionHandlerResolver getInstance() {
+        return dssRestExceptionHandlerResolver;
+    }
+
+    private final Logger logger = LoggerFactory.getLogger(DssRestExceptionHandlerResolver.class);
+    private final Map<Class<? extends DssRestActorService>, Map<Class<? extends Exception>, ExceptionHandleMethod>> exceptionHandlerMap;
+
+    private DssRestExceptionHandlerResolver() {
+        this.exceptionHandlerMap = new HashMap<>();
+    }
+
+    public void setExceptionHandlerMap(DssExceptionHandler dssExceptionHandler) {
+        Objects.requireNonNull(dssExceptionHandler);
+
+        Method[] methods = dssExceptionHandler.getClass().getMethods();
+        HashMap<Class<? extends Exception>, ExceptionHandleMethod> globalExceptionHandlerMap = new HashMap<>();
+
+        for (Method method : methods) {
+            if (method.isAnnotationPresent(ExceptionHandler.class)) {
+                ExceptionHandler annotation = method.getAnnotation(ExceptionHandler.class);
+
+                for (Class<? extends Exception> exception : annotation.exception()) {
+                    globalExceptionHandlerMap.put(exception, setExceptionHandleMethod(dssExceptionHandler, method));
+                }
```

Comment:
I'd like to share what my concern is. This is same case on lines 49~51 too.
If `setExceptionHandlerMethod()` return null, `cotainMethod()` method of `DssRestExceptionHandlerActor` class may be return true.
For example)
```java
Map<String, Integer> map = new HashMap<>();
map.put("hello", null);
System.out.println(map.containsKey("hello")); // true
```

So, I recommend that this code will be changed like below.
```java
for (Class<? extends Exception> exception : annotation.exception()) {
    ExceptionHandleMethod exceptionHandleMethod = setExceptionHandleMethod(dssExceptionHandler, method);
    
    if (Objects.nonNull(exceptionHandleMethod)) {
            globalExceptionHandlerMap.put(exception, exceptionHandleMethod);
    }
 }
```

Reply:
Thank you for review!! @Doyuni 

I think this method can't return null.

The reasons are as follows:
setExceptionHandleMethod(dssExceptionHandler, method); is return new ExceptionHandleMethod instance. 
```java
    private ExceptionHandleMethod setExceptionHandleMethod(DssExceptionHandler dssExceptionHandler, Method method) {
        return request -> {
            try {
                method.setAccessible(true);
                return (DssRestServiceResponse) method.invoke(dssExceptionHandler, request);
            } catch (IllegalAccessException | IllegalArgumentException | InvocationTargetException e) {
                logger.error("exception handling method invoke error: ", e);
            } catch (Exception e) {
                logger.error("exception handling error: ", e);
            }
            return null;
        };
    }

    // it is same as the previous code.
    private ExceptionHandleMethod setExceptionHandleMethod(DssExceptionHandler dssExceptionHandler, Method method) {
        return new ExceptionHandleMethod() {
            @Override
            public DssRestServiceResponse handlingException(DssRestServiceActorCommandRequest request) {
                try {
                    method.setAccessible(true);
                    return (DssRestServiceResponse) method.invoke(dssExceptionHandler, request);
                } catch (IllegalAccessException | IllegalArgumentException | InvocationTargetException e) {
                    logger.error("exception handling method invoke error: ", e);
                } catch (Exception e) {
                    logger.error("exception handling error: ", e);
                }
                return null;
            }
        };
    }
```

So, i think it don't need null checking

Reply:
Oh, I missed that point. 

Thank you for telling me.